.github/workflows/issue-management-update-harvester-projects.yaml

```diff
@@ -57,7 +57,8 @@ jobs:
     - name: Set Milestone to Issue
       if: ${{
         fromJSON(steps.is-harvester-member.outputs.teams)[0] != null  &&
-        github.event.issue.milestone == null
+        github.event.issue.milestone == null &&
+        steps.get-status-field.outcome == 'success'
         }}
```

Comment:
Not all issues could be assigned with Planning milestone. We only allow the issues in the Harvester sprint to be added.